evalml/pipelines/components/utils.py

```diff
@@ -98,6 +98,28 @@ def get_estimators(problem_type, model_families=None):
     return estimator_classes
 
 
+def estimator_unable_to_handle_nans(estimator_class):
```

Comment:
It may be safer to implement this as the opposite, e.g. "able_to_handle_nans". The concern I have with doing it like this is that if we add a new estimator, this function will return `False` if we forget to update this file and a false negative in this case is costly because it will mean AutoMLSearch will error out.

Not blocking - let's handle it in a follow-up.